requirements.txt

```diff
@@ -8,3 +8,4 @@ ipyparallel>=6
 toolz>=0.8
 scikit-learn>=0.18.1
 numdifftools>=0.9
+arviz>=0.12.1
```

Comment:
since arviz updates were not supposed to be included in this pull request, maybe arviz should not be included in the requirements

Reply:
this is true